test/functional/query.spec.ts

```diff
@@ -133,27 +93,16 @@ describe("NodeJS CRUD Tests", function() {
       assert.equal(docs2.length, 1, "second batch size is unexpected");
       assert.equal(docs2[0].id, resources.doc3.id, "second batch element should be doc3");
 
-      // validate Iterator.executeNext with continuation token
-      queryIterator = resources.container.items.readAll({
-        maxItemCount: 2,
-        continuation: firstResponse.continuation as string
-      });
-      const secondResponse = await queryIterator.fetchNext();
-      assert(secondResponse.requestCharge > 0, "RequestCharge has to be non-zero");
-      assert.equal(secondResponse.resources.length, 1, "second batch size with continuation token is unexpected");
-      assert.equal(secondResponse.resources[0].id, resources.doc3.id, "second batch element should be doc3");
-    };
-
-    it("nativeApi validate QueryIterator iterator toArray name based", async function() {
-      await queryIteratorToArrayTest();
-    });
-
-    it("validate queryIterator asyncIterator", async function() {
-      await queryIteratorAsyncIteratorTest();
-    });
-
-    it("nativeApi validate queryIterator iterator executeNext name based", async function() {
-      await queryIteratorExecuteNextTest();
+      // TODO: Continuation tokens are no longer returned by fetchNext because it goes down the pipelines execution context path - stfaul
```

Comment:
@christopheranderson Let's discuss this tomorrow. Now that everything goes through the pipelined query execution context, we lose continuation tokens for simple queries